### DISCUSSION TITLE
Feature request: add an option for disable exiting fullscreen gesture

### DISCUSSION THREAD

**Discussion by UlyssesZh** [Genymobile/feedback#11] [Genymotion Desktop] [answered]
I'm using Genymotion on Ubuntu 20.04 with a touchscreen. When I use Genymotion virtual device in fullscreen, I cannot swipe from top to open the status bar because it is a gesture of exiting fullscreen. Although this [can be worked around](https://superuser.com/a/1571323/1418734), I hope there would be an option for toggling this gesture in Genymotion settings (if it's possible).

**Comment by agavignet** [comment, marked as answer]
Hi @UlyssesZh,

Unfortunately this looks like an issue specific to your window manager. Genymotion does not have any control on gnome gestures. We can't give you an ETA for your suggestion.

Thank you,
Best regards
